modules/op-constructing-pipelines-using-pipeline-builder.adoc

```diff
@@ -43,7 +43,7 @@ For this example:
 ... Add an output resource. In the *Name* field, enter `Img`, and then from the *Resource Type* drop-down list, select *Image*.
 .. Optional: The *Parameters* for a task are prepopulated based on the specifications of the task. If required, use the *Add Parameters* link to add additional parameters.
 +
-.. If a *Missing Resources* warning is displayed on a task if the resources for the task are not specified. Click the *s2i-nodejs* task to see the side panel with details for the task.
+.. A *Missing Resources* warning is displayed on a task if the resources for the task are not specified. This example uses the *openshift-client* Task. Click the corresponding task to see the side panel with details for the task.
```

Comment:
Should "**openshift-client** Task" be "**openshift-client** task" in lowercase?

Reply:
@pneedle-rh I rechecked with SME, it is the the *s2i-nodejs* task that we need to mention here. So retaining the original text as is. Step 2.f.iv. of the procedure does talk about the *openshift-client* task. So we are good.